src/NexusMods.Jobs/JobContext.cs

```diff
@@ -86,8 +87,9 @@ public void SetPercent<TVal>(TVal current, TVal max) where TVal : IDivisionOpera
         _progress.OnNext(percent);
     }
 
-    public void SetProgressRate(double rate)
+    public void SetRateOfProgress(double rate)
     {
+        Debug.Write($"Rate of progress: {rate}");
```

Comment:
This may be an accidental leftover.
And if it isn't, it's missing a line ending.